recipe/run_test.sh

```diff
@@ -6,7 +6,7 @@ cp -r tests /tmp/
 
 pushd /tmp
 
-${PYTHON} -m pytest -s -rxs -v -k "not (test_fio_ls_single_layer or test_directory or test_directory_trailing_slash or test_options or test_transaction or test_encoding_option_warning or test_search_debian_gdal_data or test_search_gdal_data_debian or test_collection_zip_http or test_compound_crs or test_mapinfo or test_no_append_driver_cannot_append[PCIDSK])"  -m "not wheel" tests
+${PYTHON} -m pytest -s -rxs -v -k "not (test_fio_ls_single_layer or test_directory or test_directory_trailing_slash or test_options or test_transaction or test_encoding_option_warning or test_search_debian_gdal_data or test_search_gdal_data_debian or test_collection_zip_http or test_compound_crs or test_mapinfo or test_no_append_driver_cannot_append[PCIDSK]) or test_write_or_driver_error[DGN]"  -m "not wheel" tests
```

Comment:
Ideally we should skip it only on ARM but I would argue that we are already skipping a lot of tests and many of them are not really that useful here in the package.

Reply:
Yes, that's fine I think

Reply:
I think the test name should be before the `)`